producer/src/test/kotlin/streams/integrations/KafkaEventRouterSuiteIT.kt

```diff
@@ -17,7 +17,7 @@ import streams.utils.StreamsUtils
         KafkaEventRouterSimpleTSE::class,
         KafkaEventRouterWithConstraintsTSE::class,
         KafkaEventRouterEnterpriseTSE::class,
-        KafkaEventRouterLogCompactionTSE::class
+        KafkaEventRouterCompactionStrategyTSE::class
```

Comment:
perché hai cambiato nome?

Reply:
Perché (dalla riga 615) ho aggiunto due test che non riguardano in senso stretto il Log Compaction di kafka, ma più il `kafka.streams.log.compaction.strategy` quando è impostato a compact. In effetti pensandoci un nome più corretto sarebbe `KafkaEventRouterCompactionStrategyCompactTSE`, posso cambiarlo così, oppure rimetterlo come prima?

Reply:
no va bene